src/smt/smt_engine.cpp

```diff
@@ -3699,16 +3698,11 @@ bool SmtEnginePrivate::simplifyAssertions()
     dumpAssertions("pre-theorypp", d_assertions);
```

Comment:
Nit: this can be removed because the PreprocessingPass class takes care of the dumping/chatting/tracing.